pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -52,6 +52,8 @@
           = I18n.t(entry[:text])
       - elsif entry[:type] == "image"
         %img.image{src: entry[:url]}
+      - elsif entry[:type] == "image-tablet"
```

Comment:
Added a tablet image option for use on homepage.json: https://github.com/code-dot-org/code-dot-org/blob/ea32e656df3fef6cde8f5090039d75210d2394f7/pegasus/sites.v3/code.org/homepage.json#L93-L94